src/main/java/school/hei/haapi/endpoint/rest/security/SecurityConf.java

```diff
@@ -152,6 +151,10 @@ protected void configure(HttpSecurity http) throws Exception {
         .hasAnyRole(TEACHER.getRole(), MANAGER.getRole())
         .antMatchers(GET, "/fees")
         .hasAnyRole(MANAGER.getRole())
+        .antMatchers(GET, "/fees/types")
+        .hasAnyRole(MANAGER.getRole())
+        .antMatchers(PUT, "/fees/types/**")
+        .hasAnyRole(MANAGER.getRole())
```

Comment:
/fees/types/** covers ANYTHING after /fees/types/

such as /fees/types/not-a-valid-endpoint